spec/oas3.config.json

```diff
@@ -6,5 +6,6 @@
    "invokerPackage": "com.equinix.openapi",
    "modelPackage": "com.equinix.openapi.metal.v1.model",
    "apiPackage": "com.equinix.openapi.metal.v1.api",
-   "hideGenerationTimestamp": true
+   "hideGenerationTimestamp": true,
+   "disallowAdditionalPropertiesIfNotPresent": false
```

Comment:
Per https://openapi-generator.tech/docs/generators/java/

>  If false, the 'additionalProperties' implementation (set to true by default) is compliant with the OAS and JSON schema specifications. If true (default), keep the old (incorrect) behaviour that 'additionalProperties' is set to false by default.
>
> - **false**:  The 'additionalProperties' implementation is compliant with the OAS and JSON schema specifications.
> - **true** (default): Keep the old (incorrect) behaviour that 'additionalProperties' is set to false by default.

Sounds good to me. This should work well for this project because we can not be assured that every field returned by the API is covered by the spec today. The accessor method that will be generated will give us access to additional fields and the unknown fields won't trigger errors.
    